afl/src/bin/cargo-afl.rs

```diff
@@ -274,7 +262,7 @@ If you see an error message like `shmget() failed` above, try running the follow
 
     cargo-afl afl system-config
 
-Note: You might be prompted to enter your password as root privileges are required and hence sudo is run within this command."
+Note: You will be prompted to enter your password as root privileges are required and hence sudo is run within this command."
```

Comment:
fyi I put "might" because if the sudo user is configured with NOPASSWD or already used sudo in the shell then there will be no password prompt.